packages/hooks/src/useIsP2PEnabled.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { useStore } from '@deriv/stores';
+import { useFetch, useInvalidateQuery } from '@deriv/api';
+
+const useIsP2PEnabled = () => {
+    const { client, traders_hub } = useStore();
+    const { is_authorize, loginid, currency, is_virtual } = client;
+    const { is_low_risk_cr_eu_real } = traders_hub;
+    const invalidate = useInvalidateQuery();
+    const { data, ...rest } = useFetch('website_status', { options: { enabled: is_authorize } });
+
+    const is_p2p_supported_currency = Boolean(
+        data?.p2p_config?.supported_currencies.includes(currency.toLocaleLowerCase())
+    );
+    const is_p2p_enabled = is_p2p_supported_currency && !is_virtual && !is_low_risk_cr_eu_real;
+
+    // Todo: should replace with the next line instead once BE is fixed.
+    // const is_p2p_enabled = data?.p2p_config?.disabled === 0;
```

Comment:
We can now use `disabled` in `p2p_config` as it is already implemented by BE.

Reply:
@farzin-deriv The behavior is a bit different than what we have with the legacy code, If the user's current account is not USD we don't show the P2P tab to the users, But the flag from BE doesn't check the current account, It checks all the accounts and returns true if there are any USD accounts. 

I checked this with Bill and he said we should use the BE flag and we couldn't figure out why we are doing it differently in FE so I'm just keeping the old behavior for now 🙇🏻